.travis.yml

```diff
@@ -41,8 +41,10 @@ env:
 
 install:
 - testing/install_test_dependencies.sh
-- pecl install mongodb || /bin/true
-- phpenv config-add testing/mongodb.ini
+- if [ ! `php -r 'echo extension_loaded("mongodb");'` ]; then
+  pecl install mongodb || /bin/true;
```

Comment:
Ah, but there is still a case mongodb extension is available as a shared module.
Please just ignore the comment above.

LGTM